jspdf.js

```diff
@@ -1118,15 +1118,16 @@ var jsPDF = (function(global) {
       },
       'getTextColor': function getTextColor() {
         var colorEncoded = textColor.split(' ')
-        if (colorEncoded.length == 2 && colorEncoded[-1] == 'g') {
-          return '#000000'
-        } else {
-          var x;
-          var colorAsHex = '#'
-          for (var i = 0; i < 3; i++) {
-            x = Math.floor(parseFloat(colorEncoded[i]) * 255).toString(16);
-            colorAsHex += (x.length == 1) ? "0"+x : x;
-          }
+        if (colorEncoded.length == 2 && colorEncoded[colorEncoded.length-1] == 'g') {
```

Comment:
` if (colorEncoded.length == 2 && colorEncoded[colorEncoded.length-1] == 'g') {`

Well umm... if colorEncoded.length equals 2, then colorEncoded.length -1 equals 1

So why not instead?

` if (colorEncoded.length === 2 && colorEncoded[1] == 'g') {`

Reply:
Good point, will revise